themes/SuiteP/css/campaigns.scss

```diff
@@ -569,6 +569,11 @@ div.wizard-unique-elem .header-2 {
   margin: 5px;
 }
 
+#email_template_editor ul.mozaik-list {
```

Comment:
It is better to use a unique class instead of a id. An id really should be for a unique item. It would be nicer if you could change #email_template_editor to .email_template_editor. However, it's not a huge issue.